### PR TITLE
app - making the leaderboards more reusable; resolves #6.

### LINKED ISSUE #6: There is only one chart
We need multiple charts because we cant track 5 minute games, and 1hr games in the same spot, that wouldn't make sense

### PR DESCRIPTION
**Pull request by neverendingqs**
To be merged after #10

### CODE REVIEW

**Comment by neverendingqs** [comment]
Fixed. This is what I get for force pushing.

**Comment by neverendingqs** [comment]
Wait...

File: data/ruleset2.txt [hunk 1]
```diff
@@ -0,0 +1,6 @@
+Winner,Loser,optionalIfTie
```
Comment (turtleman270) [inline data/ruleset2.txt]: These will need to match the new format:
White, Black, winner

Comment (neverendingqs) [reply data/ruleset2.txt]: ohhhhh you merged something. i get it now lol

**Comment by neverendingqs** [comment]
Got it. Ready for re-review.

**Comment by turtleman270** [comment]
There we go, now CI is running

**Pull request merged**